### PR TITLE
LPS-196452 Use Localized Object Field Business Type

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPS-196452)

### CODE REVIEW

**Review by Luizcarlosqueiroz — APPROVED** [review]
No problem with Poshi changes :smile:

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @igor-franca, good work on this bug, I have just a small rename suggestion :smile:

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
According to the test results, I will need to update some more poshi tests.

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
hey @danvb112 ,
could you review this PR please?

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ModalAddObjectField.tsx [hunk 1]
```diff
@@ -119,20 +119,31 @@ export function ModalAddObjectField({
...
+				(objectFieldBusinessType) => {
+					if (
+						objectFieldBusinessType.businessType !== 'Relationship'
+					) {
+						return objectFieldBusinessType;
+					}
+				}
+			);
+
+			setObjectFieldBusinessTypes(objectFieldBusinessTypes);
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ModalAddObjectField.tsx:146]: Hey @igor-franca, you can place this filter directly inside the `setObjectFieldBusinessTypes` function

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @igor-franca, I have two suggestions, fell free to ask :smile:

**Comment by igor-franca** [comment]
Hey @danvb112 thanks for the feedback, these are really valuable improvements!

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
Analyzing test results

**Comment by igor-franca** [comment]
I identified that I need to update more poshi tests. Since the business types are localized now.
Therefore, the next forced push will update the following tests:

1. ObjectUpgrade#ViewObjectFieldAfterUpgrade7413U33
2. Picklist#ViewTranslatedPicklistNameOnObjectAdmin

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:test:object

**Comment by marcelabc** [comment]
ci:report:2539283400

**Comment by igor-franca** [comment]
The relevant tests are OK.
However, the object tests are not running successfully.

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:report:2539793671

**Pull request closed without merging**